zfit/constraint.py

```diff
@@ -5,7 +5,8 @@
 
 __all__ = ["nll_gaussian"]
 
-#def nll_gaussian(params, mu, sigma):
+
+# def nll_gaussian(params, mu, sigma):
```

Comment:
I would say just remove the old one? We see the changes anyway in git :-)

Reply:
True.